Userland/Services/SystemServer/Service.cpp

```diff
@@ -269,12 +251,12 @@ void Service::did_exit(int exit_code)
             break;
         default:
             dbgln("Giving up on {}. Good luck!", name());
-            return;
+            return {};
         }
         m_restart_attempts++;
     }
 
-    activate();
+    return activate();
```

Comment:
I think it is clearer to `TRY` the activate() instead of returning activate's `ErrorOr<void>`